src/helpers/createTestHarness.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Suspense } from 'react'
+
+import { RendererProps } from '../types'
+import { WrapperComponent } from '../types/react'
+
+import { isPromise } from './promises'
+
+function TestHook<TProps, TResult>({
```

Comment:
When looking at the `preact-hooks-testing-library` code I noticed they had renamed this to `TestComponent`.  I think I like that better as that is what it is.  It isn't a hook.

I'm happy to leave it, but given the size of the refactoring, it might be appropriate to rename it now.

Reply:
I just did it 😛